qemu/tests/hpt_basic.py

```diff
@@ -0,0 +1,57 @@
+import re
+import logging
+
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu HPT basic test:
+    1) Start a HPT guest on Power8 or Power9 host
+    2) Login guest and get the default HPT value
+    3) Increase HPT value
+    4) Check if guest's kernel crashes or not
+    4) Destroy the guest
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+    error_context.context("Start basic HPT test...", logging.info)
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    try:
+        error_context.context("Login HPT guest...", logging.info)
+        session = vm.wait_for_login(timeout=600)
+        get_hpt_cmd = params.get("getcommand", 
+                                 "cat /sys/kernel/debug/powerpc/hpt_order")
+        get_hpt = session.cmd_output(get_hpt_cmd, timeout=60)
+        if re.findall(params.get("error_check"), get_hpt):
+            test.cancel("Can't find the default HPT value so"
+                        "skip this test.")
```

Comment:
For line 29 ~ 32, I think that calling the function cmd_status_output() is better than cmd_output().

Reply:
Made use of cmd_status_output() for this part,thanks.